ddl/table.go

```diff
@@ -76,15 +76,16 @@ func onCreateTable(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64, _ error)
 func onCreateView(d *ddlCtx, t *meta.Meta, job *model.Job) (ver int64, _ error) {
 	schemaID := job.SchemaID
 	tbInfo := &model.TableInfo{}
+	oldTbInfo := &model.TableInfo{}
 	var orReplace bool
-	if err := job.DecodeArgs(tbInfo, &orReplace); err != nil {
+	if err := job.DecodeArgs(oldTbInfo, tbInfo, &orReplace); err != nil {
```

Comment:
Could we replace `oldTbInfo` with `oldTblInfo.ID`? We need the latest old table info. So we need to get the old table here.
And we'd better put the new parameter to the end like `job.DecodeArgs(tbInfo, &orReplace, oldTblInfoID)`. Otherwise, it may have problems during the rolling upgrade.

Reply:
understand,Done @zimulala